### ISSUE TITLE
Any option to have the calendar in RTL?

### ISSUE THREAD

**Issue by orenkatz** [HsuanXyz/ion2-calendar#188]
Hey,
Any option to have the calendar to be RTL?
Thanks,
Oren

**Comment by orenkatz** [comment]
Thank you!
Looks like the range style need a change as well.
I will try to see if I can change it.

**Comment by orelbutbul** [comment]
yes sorry I checked it only on Basic mode

**Comment by orenkatz** [comment]
Thanks, I was able to fix it by adding the following:

ion-calendar-month .primary .endSelection:after
{
  right: 0;
}

ion-calendar-month .primary .startSelection:after
{
  left: 0;
  right: auto;
}

ion-calendar-month .primary .days.startSelection:nth-child(7n):after, ion-calendar-month .primary .days.between:nth-child(7n) button.days-btn, ion-calendar-month .primary button.days-btn.is-last
{
  border-radius: 0px;
}

ion-calendar-month .primary .days.endSelection:nth-child(7n+1):after, ion-calendar-month .primary .days.between:nth-child(7n+1) button.days-btn, ion-calendar-month .primary button.days-btn.is-first
{
  border-radius: 0px;
}

**Comment by samih-dev** [comment]
hi, anyone work on this for ionic4? as it seems this is for ionic 3 - tried style overrides mentioned here but it didn't work as expected for all aspects

**Comment by NBAMj** [comment]
Yeah, it is working on the basic calendar only. Range type does not work properly either
